.readthedocs.yaml

```diff
@@ -0,0 +1,4 @@
+version: 2
+
+sphinx:
+  configuration: docs/conf.py
```

Comment:
This isn't going to build correctly, you need to add
```yaml
python:
  version: 3.8
  install:
    - method: pip
      path: .
      extra_requirements:
        - docs```I'm pretty sure.

Reply:
What I have in the files works fine for me on both `rtd` and locally..? Albeit tested against `version: 1` not `version: 2`

Reply:
If it works fine on rtd that's cool then but I would assume it wouldn't due to this being included in their configuration file. https://docs.readthedocs.io/en/stable/config-file/v2.html

Reply:
I should have added `extra_requirements` here along with 51316103c25e76c6f2f3f29cf5db9625c65cb697. You [shouldn't be using RTD config v1](https://docs.readthedocs.io/en/stable/config-file/v2.html#migrating-from-v1).

Reply:
Righto